docs/api/init.rst

```diff
@@ -53,6 +53,45 @@ Finalizes device-side rocSHMEM resources.
 Must be called before work-group completion if the work-group also called ``rocshmem_wg_init``.
 Must be called collectively by all threads in the work-group.
 
+ROCSHMEM_INIT_ATTR
+------------------
+.. cpp:function:: __host__ int rocshmem_init_attr(unsigned int flags, rocshmem_init_attr_t *attr)
+
+  :param flags: initialization method to be used.
```

Comment:
When I checked the rocm doc for another product they don't use a final period in the param description, I modified for this to be consistent in the 6.4 documentation.
```suggestion
  :param flags: initialization method to be used
```